### PR TITLE
Added a new mob and a weapon for Purpose.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Flechettes not having any AP made them kinda useless.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/projectiles/guns/energy/scythe.dm [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+	if(!istype(terminator.species, /datum/species/machine/hunter_killer))
+		to_chat(terminator, SPAN_WARNING("These systems are far too alien for even you to figure out."))
+		return
+
+	if(wielded)
+		playsound(terminator, 'sound/items/goggles_charge.ogg')
+		toggle_scope(2, terminator)
+	else
+		to_chat(terminator, "<span class='warning'>You can't look through the scope without stabilizing the rifle!</span>")
```
Comment (FluffyGhoster) [inline code/modules/projectiles/guns/energy/scythe.dm:78]: SPAN_WARNING

File: code/modules/clothing/spacesuits/rig/suits/terminator.dm [hunk 2]
```diff
@@ -1,8 +1,9 @@
-/obj/item/rig/terminator
-	name = "\improper Military Frame exoskeleton"
-	desc = "A robust synth exoskeleton outfitted with state of the art infiltration tools. Creepy."
-	icon = 'icons/clothing/rig/terminator.dmi'
-	icon_state = "terminator_rig"
+/obj/item/rig/hunter_killer
+	name = "\improper purpose combat layer"
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/terminator.dm:2]:
```suggestion
	name = "\improper Purpose combat layer"
```

File: code/modules/organs/robolimbs.dm [hunk 3]
```diff
@@ -102,8 +102,8 @@ GLOBAL_DATUM(basic_robolimb, /datum/robolimb)
 
 /datum/robolimb/terminator
 	company = PROSTHETIC_HK
-	desc = "A ludicrously expensive and EMP shielded component, these types of limbs are best suited for highly specialized cyborgs."
-	icon = 'icons/mob/human_races/ipc/r_terminator.dmi'
+	desc = "A ludicrously complex prosthetic created for Purpose Hunter-Killers."
```
Comment (SleepyGemmy) [inline code/modules/organs/robolimbs.dm:105]:
```suggestion
	desc = "A ludicrously complex prosthetic created for Purpose's Hunter-Killers."
```
optional/subjective.

File: code/modules/projectiles/guns/energy/scythe.dm [hunk 4]
```diff
@@ -0,0 +1,78 @@
+/obj/item/gun/energy/scythe
+	name = "purpose munitions scythe"
+	desc = "<span class='warning'>A seemingly innocuous metal construction the size of a human arm. Through forces unknown, it produces alloy flechettes \
+			capable of piercing even the strongest armor known to the Spur.</span>"
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/energy/scythe.dm:4]:
```suggestion
	desc = SPAN_WARNING("A seemingly innocuous metal construction the size of a human arm. Through forces unknown, it produces alloy flechettes \
			capable of piercing even the strongest armor known to the Spur.")
```

File: code/modules/projectiles/guns/energy/scythe.dm [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+	slot_flags = SLOT_BACK
+	fire_sound = 'sound/weapons/railgun.ogg'
+	safetyoff_sound = 'sound/weapons/railgun_insert.ogg'
+	hitsound = 'sound/weapons/heavysmash.ogg'
+	can_suppress = FALSE
+	has_safety = FALSE
+	has_item_ratio = FALSE
+
+	max_shots = 20
+	self_recharge = 1
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/energy/scythe.dm:20]:
```suggestion
	self_recharge = TRUE
```

File: code/modules/projectiles/guns/energy/scythe.dm [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+	reliability = 100
+	projectile_type = /obj/item/projectile/bullet/flechette
+
+	firemodes = list(
+		list(mode_name="semiauto", burst=1, projectile_type=/obj/item/projectile/bullet/flechette),
+		list(mode_name="3-round bursts", burst=3, burst_delay=ROF_SUPERHEAVY, burst_accuracy=list(2,1,1), dispersion=list(0, 10, 15), projectile_type=/obj/item/projectile/bullet/flechette),
+		list(mode_name="explosive", burst=1, projectile_type=/obj/item/projectile/bullet/flechette/explosive)
+		)
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/energy/scythe.dm:28]:
```suggestion
	)
```

File: code/modules/projectiles/guns/energy/scythe.dm (hunk 1 above)
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/energy/scythe.dm:78]:
```suggestion
		to_chat(terminator, SPAN_WARNING("You can't look through the scope without stabilizing the rifle!"))
```

File: code/modules/projectiles/guns/energy/scythe.dm [hunk 7]
```diff
@@ -0,0 +1,78 @@
+/obj/item/gun/energy/scythe
+	name = "purpose munitions scythe"
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/energy/scythe.dm:2]:
```suggestion
	name = "\improper Purpose munitions scythe"
```

File: code/modules/projectiles/guns/energy/scythe.dm (hunk 4 above)
Comment (NonQueueingMatt) [reply code/modules/projectiles/guns/energy/scythe.dm:4]: This won't compile

**Review by SleepyGemmy — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**